Recommend using a service account for BitBucket (#5302)
dbt-labs/docs.getdbt.com: website/docs/docs/cloud/git/import-a-project-by-git-url.md

## BitBucket

> You can import your BitBucket repository into dbt Cloud using a deploy key. To preserve account security, we recommend you use a service account to add the BitBucket deploy key and maintain the connection between your BitBucket repository and dbt Cloud. BitBucket attributes each repository commit and other actions to the actor's Bitbucket email address.

Author: ```suggestion
You can import your BitBucket repository into dbt Cloud using a deploy key. To preserve account security, we recommend you use a service account to add the BitBucket deploy key and maintain the connection between your BitBucket repository and dbt Cloud. BitBucket attributes each repository commit and other git actions to the actor's Bitbucket email address.
```

> You can import your BitBucket repository into dbt Cloud using a deploy key. To preserve account security, we recommend you use a service account to add the BitBucket deploy key and maintain the connection between your BitBucket repository and dbt Cloud. BitBucket attributes each repository commit and other git actions to the actor's Bitbucket email address.

Reviewer: ```suggestion
Use a deploy key to import your BitBucket repository into dbt Cloud. To preserve account security, use a service account to add the BitBucket deploy key and maintain the connection between your BitBucket repository and dbt Cloud. 

BitBucket links every repository commit and other Git action (such as opening a pull request) to the email associated with the user's Bitbucket account.
```

## BitBucket

Use a deploy key to import your BitBucket repository into dbt Cloud. To preserve account security, use a service account to add the BitBucket deploy key and maintain the connection between your BitBucket repository and dbt Cloud. 

> BitBucket links every repository commit and other Git action (such as opening a pull request) to the email associated with the user's Bitbucket account.

Reviewer: ```suggestion
BitBucket links every repository commit and other Git actions (such as opening a pull request) to the email associated with the user's Bitbucket account.
```
